Dockerfile

```diff
@@ -39,7 +39,7 @@ ENV GPU_NODE_MIN_SIZE="0"
 ENV GPU_PER_NODE="1"
 
 # gcloud config
-ENV KUBERNETES_VERSION="latest"
+ENV KUBERNETES_VERSION="1.14"
```

Comment:
Kubernetes has a really fast (compared to something like Ubuntu, which I'm familiar with) release schedule:
https://kubernetes.io/docs/setup/release/version-skew-policy/#supported-versions

The most concerning part is that a given minor release ends up only being supported for 9 months total. In the Kubernetes docs, they don't even talk about anything below 1.16 anymore.

Using a known version is desirable but, if we're going to pin it now, maybe we should try to pin to the newest version possible.

Reply:
The newest version possible (1.16.8) is incompatible with our helm installation (#345).

GKE is currently using 1.14 as the default version.